.github/workflows/build.yml

```diff
@@ -34,5 +51,6 @@ jobs:
           NODE_VERSION: ${{ matrix.version }}
         run: |
           cd $ECR_REPOSITORY
+          aws s3 cp s3://node-test-bench-app-entry-script/entry.sh entry.sh
```

Comment:
this was a good idea to do it in the build stage rather than run, but with that in mind to we even need to pull from S3?

can't we just `cp ../scripts/entry.sh .`?  You can't do that from a Dockerfile, but 🤦 no one thought to do it from a shell script in CI, which is so simple.

A few other notes:

1.  I think you're going to have to parameterize the JS script that gets passed in some way.  Some use `index.js` some use `server.js` etc - unless you already tested this with all the apps and I'm wrong
2. Beyond the scope, but I don't think it would be much more work, any reason we can't do this for the Dockerfile as well, they're all the same too I believe.